configuration/concepts/consultation_prenatale.csv

```diff
@@ -0,0 +1,51 @@
+uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1300
+c4b48a7a-7c13-4e7d-89ad-1df68c5c6801,,,First Visit,Première visite,First Visit,Première visite,,,Misc,Coded,,Yes;No,,,,,,,,,
+5bab781c-96be-4c32-b75b-d7906ddff555,,,Number of Pregnancies,Nombre de grossesses,Number of Pregnancies,Nombre de grossesses,,,Misc,Numeric,,,,FALSE,,,,,,,
+b346c76b-56ba-4c27-97cc-f7b0f85bf581,,CIEL:160080,Number of Term Pregnancies,Nombre de grossesses à terme (Parité),Number of Term Pregnancies,Nombre de grossesses à terme (Parité),,,Misc,Numeric,,,,FALSE,,,,,,,
+1b6a7d76-755c-4db9-8f05-42fe28fc3175,,CIEL:160601,Number of live births,Nombre de naissances vivantes,Number of live births,Nombre de naissances vivantes,,,Misc,Numeric,,,,FALSE,,,,,,,
+dfb79477-ab61-47c4-b308-e1ab09757c18,,CIEL:162595,Vaginal delivery,Accouchement par voie naturelle,Vaginal delivery,Accouchement par voie naturelle,,,Finding,N/A,,,,,,,,,,,
+5066f452-5d8b-4e04-b815-0c212929e47e,,CIEL:1171,Cesarean,Césarienne,Cesarean,Césarienne,,,Misc,N/A,,,,,,,,,,,
+90950144-af82-460d-9393-e5e412445075,,,No workbirth (Abortion),Aucun accouchement (Avortement),No workbirth (Abortion),Aucun accouchement (Avortement),,,Misc,N/A,,,,,,,,,,,
+2a07d3b8-c3d5-4fae-ac89-e0a1dfdd9915,,CIEL:161033,Birth outcome during the last pregnancy,Issue de l'accouchement précédent,Birth outcome during the last pregnancy,Issue de l'accouchement précédent,,,Misc,Coded,,Vaginal delivery;Cesarean;No workbirth (Abortion),,,,,,,,,
+df9cc02b-b427-4262-b30a-7b973aef23fc,,CIEL:1536,Home,Domicile,Home,Domicile,,,Misc,N/A,,,,,,,,,,,
+023e0a9e-cf47-4a4d-9cf4-b2edfbe7fc33,,CIEL:1572,Place of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,Place of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,,,Misc,Coded,,St Jean Limbe Hospital;Good Samaritain Limbe Hospital;Justinien Cap Haitien Hospital;Sacred Heart Milot Hospital;HBCH Hospital;Gable Hospital;Borgne Hospital;Trou du Nord Hospital;Home;Other,,,,,,,,,
+05306cba-374b-40e8-ac40-4e8a510add61,,,Place of delivery of the last pregnancy data,Lieux d'accouchement des dernières données de grossesse,Place of delivery of the last pregnancy data,Lieux d'accouchement des dernières données de grossesse,,,ConvSet,N/A,,,Place of delivery of the last pregnancy; Other Health Centre,,,,,,,,
+50a37eff-81a3-4cef-a19c-1f85d4e690d6,,CIEL:1596,Date of most recent pregnancy delivery,Date du dernier accouchement,Date of most recent pregnancy delivery,Date du dernier accouchement,,,Misc,Date,,,,,,,,,,,
+e3892a91-1428-4816-8e83-834dcd738723,,,Any Previous Delivery Complications,,Any Previous Delivery Complications,Toute complication de livraison précédente,,,Misc,Coded,,Yes; No,,,,,,,,,
```

Comment:
Missing the FSN in fr?
"Any Previous Delivery Complications" seems a bit too much like a sentence already.
Should probably be more "Previous Delivery Complications".

In fr: "Complications lors du dernier accouchement"

Reply:
"Previous Delivery Complications" is on line 15

Reply:
Could Line 15 be "Previous Delivery Complication Details"?

Reply:
Done